pom.xml

```diff
@@ -139,17 +139,12 @@
     <maven.version>3.6.3</maven.version>
     <dropwizard.version>4.1.14</dropwizard.version>
     <micrometer.version>1.6.0</micrometer.version>
+    <findbugs.jsr305.version>3.0.2</findbugs.jsr305.version>
 
-    <hadoop2.7.version>2.7.7</hadoop2.7.version>
-    <hadoop3.0.version>3.0.3</hadoop3.0.version>
-    <hadoop3.1.version>3.1.3</hadoop3.1.version>
     <hadoop3.2.version>3.2.4</hadoop3.2.version>
     <hadoop3.3.version>3.3.6</hadoop3.3.version>
-    <hadoop.version>${hadoop2.7.version}</hadoop.version>
+    <hadoop.version>${hadoop3.3.version}</hadoop.version>
```

Comment:
Hadoop 3.2 is EOL, from the community perspective, we'd better choose a supported version.
https://lists.apache.org/thread/ybygrhvqok0f44s7yzm6c24n0b0s727s

Reply:
another reason is, as we use the hadoop-minicluster to launch Mini YARN cluster for testing in some modules, while since Spark 3.2.0, the pre-built Spark tgz ships Hadoop 3.3.x jars, Hadoop requires that client version <= server version, so we must use Hadoop 3.3 to launch Mini YARN cluster to allow Spark works properly.